docs/pages/branding/about.tsx

```diff
@@ -19,82 +19,266 @@ import BrandingBeginToday from 'docs/src/modules/branding/BrandingBeginToday';
 import BrandingDiscoverMore from 'docs/src/modules/branding/BrandingDiscoverMore';
 import BrandingPersona from 'docs/src/modules/branding/BrandingPersona';
 
+function BrandingTeam() {
+  return (
+    <Box sx={{ bgcolor: 'greyF3', pt: 15, pb: 10 }}>
+      <Container sx={{ position: 'relative' }}>
+        <Box
+          component="img"
+          src="/static/branding/block4.svg"
+          alt=""
+          sx={{
+            width: 207,
+            height: 171,
+            position: 'absolute',
+            right: { xs: 10, md: 0 },
+            bottom: -175,
+          }}
+        />
+        <Typography variant="h2">Team</Typography>
+        <Typography sx={{ mt: 1.5, mb: 7, maxWidth: 700 }}>
+          Material-UI is maintained by a group of invaluable core contributors, with the massive
+          support and involvement of the community.
+        </Typography>
+      </Container>
+    </Box>
+  );
+}
+
+const company = [
+  {
+    name: 'Olivier Tassinari',
+    src: '/static/branding/about/olivier.jpg',
+    title: 'Co-founder',
+    location: 'Paris, France',
+    twitter: 'olivtassinari',
+    github: 'oliviertassinari',
+  },
+  {
+    name: 'Matt Brookes',
+    src: '/static/branding/about/matt.jpg',
+    title: 'Co-founder',
+    location: 'London, UK',
+    twitter: 'randomtechdude',
+    github: 'mbrookes',
+  },
+  {
+    name: 'Sebastian Silbermann',
+    src: '/static/branding/about/sebastian.jpg',
+    title: 'Community focus',
+    location: 'Dresden, Germany',
+    twitter: 'sebsilbermann',
+    github: 'eps1lon',
+  },
+  {
+    name: 'Damien Tassone',
+    src: '/static/branding/about/damien.jpg',
+    title: 'Enterprise focus',
+    location: 'Lodon, UK',
```

Comment:
```suggestion
    location: 'Barcelona, Spain',
```

Reply:
@dtassone I have updated the location back. It can be important when local dev events are run, or simply to be more accurate.